code/controllers/subsystem/processing/processing.dm

```diff
@@ -1,40 +1,65 @@
 //Used to process objects. Fires once every second.
 
-var/datum/subsystem/processing/SSprocessing
 /datum/subsystem/processing
 	name = "Processing"
-	priority = 25
-	flags = SS_BACKGROUND|SS_POST_FIRE_TIMING|SS_NO_INIT
-	wait = 10
+	can_fire = FALSE
 
 	var/stat_tag = "P" //Used for logging
-	var/list/processing = list()
-	var/list/currentrun = list()
+	var/list/processing_list = list()	//what's processing
+	var/list/run_cache = list()	//what's left to process in the next run
+	var/delegate	//what the processing call is
 
 /datum/subsystem/processing/New()
-	NEW_SS_GLOBAL(SSprocessing)
+	if(type == /datum/subsystem/processing)
+		flags |= SS_NO_FIRE	//this SS should be derived, but MC will create it anyway
 
-/datum/subsystem/processing/stat_entry()
-	..("[stat_tag]:[processing.len]")
+/datum/subsystem/processing/stat_entry(append, forward = FALSE)
+	if(forward)
+		..(append)
+	else if(processing_list)
+		..("[stat_tag]:[processing_list.len][append]")
+	else
+		..("[stat_tag]:FIX THIS SHIT")
 
-/datum/subsystem/processing/fire(resumed = 0)
+/datum/subsystem/processing/proc/start_processing(datum/D)
+	if(D)
+		processing_list[D] = D
+		can_fire = TRUE
```

Comment:
I'm going to point out that this was how it used to be done, I changed it to use a flag on the object because it made a noticeable change in world init. 

the idea was also that START_PROCESSING was for the **main** processing of an object, not any subsystem processing of that object, so it also prevented putting something in both ssfast and ssobj. the first would win out.